### PR TITLE
Add action for when system tools are run

### PR DESCRIPTION
**Pull request by jessepearson**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Adds an action to tell when tools are run from WooCommerce > Status > Tools, just like what exists for if the tools are run [via the rest API](https://github.com/woocommerce/woocommerce/blob/master/includes/api/class-wc-rest-system-status-tools-controller.php#L259-L287). 

### How to test the changes in this Pull Request:

1. Tie into the action
2. Run a tool

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Add action for when system tools are run.

### CODE REVIEW

File: includes/admin/class-wc-admin-status.php [hunk 1]
```diff
@@ -51,6 +51,22 @@ public static function status_tools() {
 			} else {
 				echo '<div class="error inline"><p>' . esc_html( $response['message'] ) . '</p></div>';
 			}
+
+			$tool = $tools[ $action ];
```
Comment (claudiulodro) [inline includes/admin/class-wc-admin-status.php]: As seen on line 40, there is the potential of the tool not existing for the input action. This should gracefully handle that somehow. The current implementation will cause PHP errors.

Best solution is probably to move this new code inside the `if` block on line 40 and fire the 'woocommerce_system_status_tool_executed ' hook right after the tool is executed. That would also closely match [this one](https://github.com/woocommerce/woocommerce/blob/master/includes/api/class-wc-rest-system-status-tools-controller.php#L259-L287) which only fires when there is a valid tool.

**Review by claudiulodro — CHANGES_REQUESTED** [review]
This seems like a reasonable change; just a tweak needed to prevent PHP errors.

**Comment by jessepearson** [comment]
@claudiulodro Updated, moved the code into the if block.

**Review by claudiulodro — APPROVED** [review]
Looks good. 👍

**Pull request merged**